### ISSUE TITLE
Add option to change speed of preview text in pop-up

### ISSUE THREAD

**Issue by VibrantTreeStump** [TeamNewPipe/NewPipe#8197]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

Sometimes the title of a video is so long that it can't be displayed all at once. When pressing and holding on the video the title runs from right to left in the pop-up but very slowly.
It would be nice if the speed at which the text moves would be customizable.

### Why do you want this feature?

Because it would be nice to be able to read titles at my natural reading pace.

**Comment by absurdlylongusername** [comment]
I don't think we should have a variable for the speed at which the text moves, but rather we should improve our UX so users can immediately see the names of videos with long text.

Either by long clicking on it to show the full name in a tooltip, or similar.
